cards.tex

```diff
@@ -140,6 +140,19 @@ \subsection{Damage Types\hfill\Reference{CRB}{451}}
 
 \end{multicols}
 
+\section{Abbreviations}
+
+\begin{tabular}{ll}
+    \Reference{CRB}{}        & Core rule book \\
+    \HitPoints{}             & Hit points \\
+    \BrokenThreshold{}       & Broken threashold \\
+    DC                       & Difficulty class \\
+    AC                       & Armor class \\
+    prof                     & Proficiency \\
+    str                      & Strength \\
+    int                      & Intelligence \\
```

Comment:
for completeness sake add all [six ability scores](https://2e.aonprd.com/Rules.aspx?ID=67)

```suggestion
    str                      & Strength \\
    dex                      & Dexterity \\
    con                      & Constitution \\
    int                      & Intelligence \\
    wis                      & Wisdom \\
    cha                      & Charisma \\
```